gnmi_server/client_subscribe.go

```diff
@@ -195,7 +195,11 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 		c.polled = make(chan struct{}, 1)
 		c.polled <- struct{}{}
 		c.w.Add(1)
-		go dc.PollRun(c.q, c.polled, &c.w, c.subscribe)
+		if strings.Contains(target, "APPL_DB") {
```

Comment:
>strings.Contains [](http://example.com/codeflow?start=5&length=16)

too relaxing condition, please be more specific, check the single-asic or multi-asic APPL_DB only.

Reply:
Done